protocol/0053-PERP-product_builtin_perpetual_future.md

```diff
@@ -83,62 +83,16 @@ Every time a [mark to market settlement](./0003-MTMK-mark_to_market_settlement.m
 
 When the `settlement_schedule` event is received we need to calculate the funding payment. Store the current vega time as `funding_period_end`.
 
-If there are no oracle data points with a timestamp less than `funding_period_end` available then funding payment is skipped and `funding_period_start` gets overwritten with `funding_period_end`.
+Skip the funding payment calculation (set payment to `0`) if any of the following conditions is met:
 
-If such points are available then the calculations discussed in the following subsections get executed and funding payments get exchanged.
+- no spot (external) data has been ingested since market was created,
+- market has been in auction throughout the entire funding period.
```

Comment:
Does this imply we should discard external data points if the market is still in opening auction, or simply skip the funding payment if the market is in opening auction? Do we still factor in the data for margin calculation?

Reply:
Let me know if the latest commit clarifies it.